src/main/java/org/cyclops/integrateddynamics/core/client/gui/container/GuiPartSettings.java

```diff
@@ -145,6 +157,16 @@ protected int getBaseXSize() {
         return 214;
     }
 
+    @Override
+    protected int getBaseYSize() {
+        return 189;
+    }
+
+    protected void updateExposeCapabilities(boolean newValue) {
+        exposeCapabilities = newValue;
+        buttonExposeCapabilities.displayString = newValue ? trueStr : falseStr;
```

Comment:
I don't really like buttons that toggle that update their label like this. A checkbox or an actual visual toggle-able button would be better. (perhaps similar to how the new mechanical machines have a toggle to auto-export fluids)

Reply:
I don't see a way to make the auto-export button style not look weird here. And isn't this how the in-game config buttons work?

Reply:
Something based on a checkbox should be doable, possibly using an image-based button using that green checkmark in `Images`.